.ci/pipeline/release_jjb.yaml

```diff
@@ -0,0 +1,65 @@
+- job-template:
+    name: "{jjb_proj}"
+    project-type: pipeline
+    folder: libxlio
+    properties:
+        - github:
+             url: "https://github.com/Mellanox/libxlio"
+        - build-discarder:
+            days-to-keep: 120
+            num-to-keep: 20
+        - inject:
+            keep-system-variables: true
+            properties-content: |
+              jjb_proj={jjb_proj}
+    description: Do NOT edit this job through the Web GUI !
+    concurrent: false
+    parameters:
+        - string:
+            name: "release_tag"
+            default: ""
+            description: "Tag to release"
+        - validating-string:
+            name: "revision"
+            default: "1"
+            regex: ^[0-9]+$
+            msg: "Revision must be an integer ONLY"
+            description: "Release revision"
+        - string:
+            name: "release_folder"
+            default: "/auto/sw/release/sw_acceleration/xlio"
+            description: "Folder to release packages into"
+        - bool:
+            name: "do_release"
+            default: true
+            description: "Release build packges into the release folder, set to false for debugging"
+        - validating-string:
+            name: "notification_email"
+            default: "a9b23b5c.NVIDIA.onmicrosoft.com@amer.teams.ms"
+            regex: ^[\w-\.]+@([\w-]+\.)+[\w-]{{2,4}}$
+            msg: "Please provide a valid email address"
+            description: "Email to send report to upon success/failure"
+        - string:
+            name: "conf_file"
+            default: ".ci/pipeline/release_matrix_job.yaml"
+            description: "Regex to select job config file. Do not change it"
+    pipeline-scm:
+        scm:
+            - git:
+                url: "{jjb_git}"
+                credentials-id: 'b7d08ca7-378c-45d6-ac4b-3f30bdf49168'
+                branches: ['tags/$release_tag']
```

Comment:
not sure how this is working for you what if i want to test my PR, i need to be able to put my commit id here
we should probably use sha1 (like all other jobs) which can take the default of the tag parameter